includes/admin/class-wc-rest-payments-terminal-locations-controller.php

```diff
@@ -162,7 +162,27 @@ public function get_store_location( $request ) {
 
 			return rest_ensure_response( $this->extract_location_fields( $location ) );
 		} catch ( API_Exception $e ) {
-			return rest_ensure_response( new WP_Error( $e->get_error_code(), $e->getMessage() ) );
+			$error = new WP_Error( $e->get_error_code(), $e->getMessage() );
+			// Stripe will return a 400 for incorrect city, state, or country.
```

Comment:
Please compact the comment into 2-3 lines if possible.

Reply:
Sure, done in 0c80f6d.

Reply:
Thanks!